README.md

```diff
@@ -56,8 +56,90 @@ Clean up the Kubernetes/OpenShift cluster:
 ```shell script
 $ make cluster/clean
 ```
+### Deploy the Operator in cluster
 
-## VPC Peering 
+Build and push the image
+```shell script
+$ make image/push IMAGE_ORG=<your quay org>
+```
+
+Update the `deploy/operator.yaml` to point at the newly built image
+```
+      ...
+        - name: cloud-resource-operator
+          image: quay.io/<your quay org>/cloud-resource-operator:v0.7.1
+          command:
+          - cloud-resource-operator
+          imagePullPolicy: Always
+      ...
+```
+
+Seed the Kubernetes/OpenShift cluster with required resources:
```

Comment:
We could add a Kubernetes/Openshift 4+ cluster as a requirement, and then just refer to this as a cluster onwards